src/commands/apphosting-repos-create.ts

```diff
@@ -0,0 +1,23 @@
+import { Command } from "../command";
+import { Options } from "../options";
+import { needProjectId } from "../projectUtils";
+import requireInteractive from "../requireInteractive";
+import { createGitRepoLink } from "../apphosting";
+import { ensureApiEnabled } from "../gcp/apphosting";
+import { APPHOSTING_TOS_ID } from "../gcp/firedata";
+import { requireTosAcceptance } from "../requireTosAcceptance";
+
+export const command = new Command("apphosting:repos:create")
```

Comment:
qq: do we expect users to use this command?

If it's just for us maybe it should be behind an experiment?

Reply:
It's hidden under "    if (experiments.isEnabled("internaltesting")) {"